### DISCUSSION TITLE
Will the NuGet packages ever move to MIT-licensing?

### DISCUSSION THREAD

**Discussion by FLindqvist** [microsoft/sqlmanagementobjects#116] [General]
Even though the source repo is MIT-licensed, the NuGet packages are still under the closed-source, making them somewhat difficult to use in some scenarios. At the same time, this repo isn't particularly easy to build from source.

Will the NuGet packages ever move to MIT licensing, or are there any particular reasons why not?

**Comment by shueybubbles** [comment]
the newer packages are MIT

**Comment by shueybubbles** [comment]
@FLindqvist   - What part of building from source do you find most challenging? The rapid evolution of the Dotnet SDK from version 5 to version 7 has caused some pain for sure, but afaik if you have Visual Studio community or a modern Dotnet SDK you can just `dotnet build dirs.proj` or `msbuild dirs.proj` in the root folder.
That's pretty much what the github  action does: https://github.com/microsoft/sqlmanagementobjects/blob/74e1ae673559f37eb016b4623233ced83e9e564e/.github/workflows/pr-validation.yml#L17
